rpm/sofia-sip.spec

```diff
@@ -1,17 +1,21 @@
 Name:       sofia-sip
 
 Summary:    Sofia SIP User-Agent library
-Version:    1.12.11
-Release:    2
-Group:      Communications/Telephony and IM
+Version:    1.13.4
+%global ver 1.13
```

Comment:
Looks not so scalable: wildcards in `%files` or lua macro can solve this.

Reply:
I agree. Lua is common and not "fancy". Wildcard would be also ok.

Reply:
I would prefer wildcards, we use those in different specs already and I find those even more simple than lua.

Reply:
> Looks not so scalable: wildcards in `%files` or lua macro can solve this.

It's perfectly scalable. This macro will be expanded as many times as necessary, just like lua macros. And one doesn't have to be familiar with lua in order to understand what it does. Spec file doesn't need to be a riddle.

Remember that the version tag is being messed with by OBS, I wouldn't make any assumptions about its contents during the build. The fewer assumptions the better. Upstream update is a manual process and there's no way one can possibly forget to update this line when changing the line above, especially when those two lines are carefully aligned like this.

I'm sure this kind of solution provides the best possible reliability/complexity ratio :)

Reply:
If you simply use wildcards in `%files` then you don't need to always modify this copy-pasted `ver` on every package version bump.

Reply:
> If you simply use wildcards in `%files` then you don't need to always modify this copy-pasted `ver` on every package version bump.

Oh, that does make sense. Done!